.kitchen.yml

```diff
@@ -16,7 +16,20 @@ platforms:
         cpus: 2
         memory: 4096
 
+
 suites:
+  - name: alibaba_automate
+    run_list: "marketplace_image::default"
+    attributes:
+      marketplace_image:
+        alibaba:
+          automate:
+            enabled: true
+      packman:
+        version: 1.0.4
+        checksums:
+          "1.0.4": 646da085cbcb8c666474d500a44d933df533cf4f1ff286193d67b51372c3c59e
```

Comment:
While I think it's good that we're upgrading packer, I'd prefer to see the default version updated to 1.0.4 and breaking changes in other packer definitions fixed. We're tracking that work on: [CLOUD-261](https://chefio.atlassian.net/browse/CLOUD-261) and [CLOUD-262](https://chefio.atlassian.net/browse/CLOUD-262)

Reply:
Agreed - did not want to assume upgrading the default version whilst I was getting the initial Alibaba work going